src/lib/components/Body/test/MultipleMarks.test.js

```diff
@@ -0,0 +1,92 @@
+import React from "react";
```

Comment:
Is this file still needed?

Reply:
The first test is a real bug that we patched in this commit  so it should stay I think.   The convertMarks stuff can go

Reply:
Removed convertMarks and related unused tests